src/main/java/io/quarkus/gizmo/AnnotationUtils.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.gizmo;
+
+import org.jboss.jandex.AnnotationValue;
+import org.objectweb.asm.AnnotationVisitor;
+
+public final class AnnotationUtils {
```

Comment:
This should probably be package private.

Reply:
@stuartwdouglas done. I made both class and method package-private. Is that OK or should only be a class?

Reply:
Perfect.